package.json

```diff
@@ -19,20 +19,22 @@
     "url": "https://github.com/qlik-oss/sn-scatter-plot.git"
   },
   "files": [
-    "assets",
-    "dist"
+    "api-specifications",
+    "dist",
+    "core",
+    "sn-scatter-plot.js",
+    "sn-scatter-plot.qext"
```

Comment:
why core?

"sn-scatter-plot.js", "sn-scatter-plot.qext" are no longer needed (except to supress the incorect warning in nebula.js)
but we may want to add the extension folder (sn-scatter-plot-ext?) if we want that to be a part of the published package

Reply:
The core package is to expose the "light-weight" build without picasso, react etc bundled inside. So it can be brought in and build with those as peer dependencies.

you are right about `sn-scatter-plot.js` and `sn-scatter-plot.qext`, I forgot about those, will update and change to the `sn-scatter-plot-ext` folder instead.